Chummer/data/weapons.xml

```diff
@@ -14755,6 +14755,62 @@
       <useskill>Exotic Ranged Weapons</useskill>
       <weapontype>firefighting cannons</weapontype>
     </weapon>
+    <weapon>
+      <id>828021e1-a9ff-44e4-9559-918aa156e279</id>
+      <name>Ruhrmetall SFW-30</name>
+      <category>Heavy Machine Guns</category>
+      <type>Ranged</type>
+      <conceal>10</conceal>
+      <accuracy>6(8)</accuracy>
```

Comment:
Don't put parentheses in the data file, the Smartgun system accessory is already going to do that automatically.

Reply:
fixed